cfg/aws/aws_sdk_logging_test.go

```diff
@@ -0,0 +1,48 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: MIT
+
+package aws
+
+import (
+	"testing"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/influxdata/wlog"
+)
+
+func TestSetSDKLogLevel(t *testing.T) {
+	cases := []struct {
```

Comment:
On the one hand, I like this parameterized testing, but on the other hand, it doesn't make the code paths for the SDK log level configuration crystal clear with isolated test paths. Not sure where I land on this atm. Gonna keep it in mind and see if I develop a strong opinion.